components/shopify_developer_app/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+By creating a custom app on Shopify, you will be able to configure the exact scopes that you require to build the workflows that you need.
+
+# Getting Started
+
+To get started, you will need to create a custom Shopify app in the Shopify Admin UI. The steps are outlined below:
+
+## Create a Shopify app
+1. Sign in to your Shopify admin account, and navigate to the store that you are managing. 
+2. Click **Develop apps**.
+3. Click **Allow custom app development**.
+4. Click **Create an App**, and name the app "Pipedream"
+
+  ![Create an App](https://res.cloudinary.com/dpenc2lit/image/upload/v1688060015/Screenshot_2023-06-29_at_10.11.43_AM_unkom4.png)
+
+## Configure Admin API scopes
```

Comment:
are all users who use the current shopify app also admins? is this going to limit access at all?

Reply:
They have to be one of the following account [types](https://help.shopify.com/en/manual/apps/app-types/custom-apps):
- Admin
- Collaborator
- Staff